### DISCUSSION TITLE
Permanent MFA enabled?

### DISCUSSION THREAD

**Discussion by the-mace** [philosowaffle/peloton-to-garmin#626] [Q&A]
Hi folks, has anyone seen this on their garmin account?
"*Permanent two-step verification has been enabled."
I noticed that one feature I enabled (I think it was for pulse/afib) said MFA was required to enable that, could that be what forced my account to MFA?
It makes automatic syncing not automatic which is annoying.

**Comment by geoffbon** [comment]
Yes, if you enable ECG then it forces two-step verification / MFA.  There's an issue which addresses this ([585](https://github.com/philosowaffle/peloton-to-garmin/issues/585)) which was making its way through the process and I see that it's signed off, which is awesome!  Looking forward to the release when it happens.
